README.md

```diff
@@ -1 +1,34 @@
-# Hello-Microverse
+<h1>Hello Microverse</h1>
+    *First microverse project submission.*
+
+![an image of the Hello World webpage](https://github.com/hamid-murambiwa/README-IMG/blob/main/Screenshot%202021-08-10%20at%2015.42.46.png)
+
+
+<h2>Built With</h2>
+*HTML 
+*CSS3
+
+<h2>Authors</h2>
```

Comment:
I strongly recommend you additional instructions about how to set up your repository in a local environment, for instance:

### Install

To get a local copy up and running follow these simple example steps.
- Open terminal
- Clone this project by the command `git clone URL/repository_name.git`
- `cd repository_name` folder
- Open `index.html` in your local browser or using Live Server in Visual Studio Code.

- [ ] [OPTIONAL] Add more instructions about your project.